content/guides/vercel-sdk-migration.md

```diff
@@ -0,0 +1,233 @@
+---
```

Comment:
also, can we mention the use of https://github.com/neondatabase/serverless/blob/main/CONFIG.md#poolqueryviafetch-boolean?

Reply:
Have added `poolQueryViaFetch` and `wsProxy` now. Skipping rest of the options, as it would increase the complexity of the guide.